### ISSUE TITLE
Basic indentation/syntax errors in 3.6?

### ISSUE THREAD

**Issue by ColPanic** [savon-noir/python-libnmap#91]
After fixing 3 previous indentation issues, here's the latest:

NunYaBusiness:TheSurg30n Todd3$ python3 Th3surg30n.py cosi_range.xml 
Traceback (most recent call last):
  File "Th3surg30n.py", line 24, in <module>
    from libnmap.parser import NmapParser, NmapParserException
  File "/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/libnmap/parser.py", line 8, in <module>
    from libnmap.objects import NmapHost, NmapService, NmapReport
  File "/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/libnmap/objects/__init__.py", line 3, in <module>
    from libnmap.objects.report import NmapReport
  File "/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/libnmap/objects/report.py", line 96
    rval = self.['_nmaprun']['startstr']

I have to ask, does this work at all with 3.6?   There are numerous indentation issues with the module, especially at the @properties and the def of things.  Or am I just doing something silly?  (On osX)

**Comment by snovvcrash** [comment]
Noticed that as well. For example, `/libnmap/objects/report.py` has three lines with different types of indention mixed: [97](https://github.com/savon-noir/python-libnmap/blob/eaf79cbfe8c60153ef4ed7d77c1bde2cda80cd54/libnmap/objects/report.py#L97), 98, 99, 100 (just blank line with TABs here).

**Comment by savon-noir** [comment]
easiest to get it fixed fast: pull request when you notice this :) i'll merge

**Comment by savon-noir** [comment]
this shall be fixed following code reformatting/linting/flake8ing. will be packaged in the v0.7.1 release and will be merged tomorrow in master branch via PR https://github.com/savon-noir/python-libnmap/pull/110
Should you notice other indent issues please add some comments in this issue. thanks!

**Comment by savon-noir** [comment]
fixed by new release [v0.7.2](https://github.com/savon-noir/python-libnmap/releases/tag/v0.7.2). 

Package is also available in pip:

```
ronald@brouette:~/dev$ pip3 install python-libnmap==0.7.2
Collecting python-libnmap==0.7.2
  Downloading python-libnmap-0.7.2.tar.gz (36 kB)
...
...
Successfully installed python-libnmap-0.7.2
ronald@brouette:~/dev$ 


```
